pinot-segment-local/src/main/java/org/apache/pinot/segment/local/indexsegment/mutable/MutableSegmentImpl.java

```diff
@@ -569,6 +569,19 @@ private RecordInfo getRecordInfo(GenericRow row, int docId) {
     return new RecordInfo(primaryKey, docId, null);
   }
 
+  @Override
+  public void getPrimaryKey(int docId, PrimaryKey reuse) {
```

Comment:
(minor) move this method next to `getRecord()` for readability

Reply:
done